ledger/ledger-api-test-tool/conformance.bzl

```diff
@@ -25,9 +25,7 @@ def conformance_test(
         timeout = "long",
         client = "//ledger/ledger-api-test-tool",
         client_args = test_tool_args + ["localhost:%s" % port for port in ports],
-        data = extra_data + [
-            "//ledger/test-common:dar-files",
```

Comment:
By default, the DAR files are bundled with the test tool binary as resources, they don't need to be added as data here.